libs/blocks/global-navigation/features/search/gnav-search.js

```diff
@@ -10,12 +10,9 @@ import {
 import { replaceKeyArray } from '../../../../features/placeholders.js';
 import { getConfig } from '../../../../utils/utils.js';
 import { debounce } from '../../../../utils/action.js';
+import getServiceConfig from '../../../../utils/service-config.js';
 
 const CONFIG = {
-  suggestions: {
-    scope: 'adobecom',
-    apiKey: 'adobedotcom2',
```

Comment:
these are not really secrets, I think we can suppress the kodiak alerts cc @salonijain3

Reply:
Ideally I still don't think we should be exposing them. It makes it easy for some third party to send requests to that endpoint and causing issues.